test/functional/zmq_test.py

```diff
@@ -24,7 +48,7 @@ def setup_nodes(self):
         except ImportError:
             raise SkipTest("python3-zmq module not available.")
 
-        # Check that bitcoin has been built with ZMQ enabled
+        # Check that bitcoin has been built with ZMQ enabled.
         config = configparser.ConfigParser()
         if not self.options.configfile:
             self.options.configfile = os.path.dirname(__file__) + "/../config.ini"
```

Comment:
Likely a pre-existing issue, on my box this prevents `python zmq_test.py` running in isolation (works through test_runner because the option already exists). If you wrap it e.g. `os.path.abspath(os.path.dirname(__file__))` it works.

Reply:
It works for me 🙄

Reply:
I'll check with your suggestion.

Reply:
Also should use `os.path.join()`

Reply:
Ok.